councilmatic_core/templates/partials/legislation_text.html

```diff
@@ -18,6 +18,19 @@ <h3>
     </iframe>
 
     <div class="divider"></div>
+{% elif legislation.html_text %}
+    {% if legislation.html_text|length > 65 %}
+    <h3>
+        <i class='fa fa-fw fa-file-text-o'></i>
+        Legislation text
+    </h3>
+    <div class='panel panel-default'>
+        <div class='panel-body'>
+            {{ legislation.html_text|safe }}
+        </div>
+    </div>
+    <div class="divider"></div>
+    {% endif %}
```

Comment:
This looks good.

I made a custom template for NYC: https://github.com/datamade/nyc-council-councilmatic/blob/master/nyc/templates/partials/legislation_text.html

Hence, a couple questions:

(1) Why is 65 the lower boundary for html length? Could one have valid html that's, say, 50 characters?
(2) Not directly related to your change, but immediately related to this template: The `ocr_full_text` is always [plain_text](https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/management/commands/import_data.py#L922). If we want this template to be reusable, then would you consider adding [the custom filter](https://github.com/datamade/nyc-council-councilmatic/blob/master/nyc/templatetags/nyc_extras.py#L10) from NYC to django-councilmatic? I do not think django's safe filter will accomplish very much.

Reply:
Yeah, maybe I ought to revert this change and just handle this in the version I am working on. The HTML that I am getting from the source is pretty OK and I feel like all I really need is the `safe` filter.

Reply:
It would great to make this partial reusable across Councilmatics. (That's not really the point of this PR, though....but it would save you and me and others the hassle of creating a custom one.)

Reply:
I feel like it's a little tricky, however, because you have to have a pretty decent amount of confidence that the HTML you will be displaying will look OK and not bork the rest of the page. Which makes me feel like, however annoying, you'll have to handle it on a case by case basis anyways.

Reply:
Yes, that's true. We can keep it as a customizable template...go ahead and revert this change.